zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/web/rest/CatalogiResource.kt

```diff
@@ -52,11 +53,11 @@ class CatalogiResource(
         return ResponseEntity.ok(zaakObjectTypes)
     }
 
-    @GetMapping("/v1/case-definition/{caseDefinitionName}/zaaktype/roltype")
+    @GetMapping("/v1/case-definition/{caseDefinitionId}/zaaktype/roltype")
```

Comment:
When you need a case definition id you should use something like `/case-definition/{caseDefinitionKey}/version/{versionTag}` and construct the id in the method.